components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -301,6 +303,11 @@ public class OAuth2Util {
      */
     public static final String APPLICATION_ACCESS_TOKEN_EXP_TIME_IN_MILLISECONDS = "applicationAccessTokenExpireTime";
 
+    /**
+     * FIdp Role Based authentication application config.
+     */
+    public static final String FIDP_ROLE_BASED_AUTHZ_APP_CONFIG = "FIdPRoleBasedAuthzApplications.AppName";
```

Comment:
unused

Reply:
This is used in places where we check the for FIdp role based authentication enabled or not eg `JDBCPermissionBasedInternalScopeValidator.java`, `RoleBasedInternalScopeValidator.java`